### ISSUE TITLE
Add new deployment type ECS Fargate

### ISSUE THREAD

**Issue by andresionek91** [microcks/microcks#1372]
### Describe the bug

Hi! I just deployed Microcks internally on ECS Fargate (not on K8s). I used CDK for the infrastructure as code. Would it be interesting for the community if I share the constructs as a library and add it to the docs?

**Comment by lbroudoux** [comment]
Hello!

It would be awesome! We got a specific `community` repository on GitHub and thought it would be the recipient of this kind of contribution. We could initialize a `/install` folder in it and then have you push a `aws-ecs-fargate` subfolder with all the content and detailed doc. 

Of course, it could be advertised on https://microcks.io via a dedicated documentation page or even a blog post 😉 

Let me know if it matches what you have in mind and it'll be my pleasure to quickly initialize stuff.

Cheers!

**Comment by andresionek91** [comment]
Yep! That's perfect! I'm going to work on that on the coming weeks!

**Comment by lbroudoux** [comment]
It is now ready here: https://github.com/microcks/community/tree/main/install 😉

**Comment by lbroudoux** [comment]
Hey @andresionek91! Any chance you got free cycles to have a look at this?
